bentoml/_internal/cli/yatai.py

```diff
@@ -1,25 +1,29 @@
 import click
 
-from bentoml._internal.yatai_rest_api_client.config import add_context, YataiClientContext, default_context_name
-
-from bentoml._internal.cli.click_utils import _echo, CLI_COLOR_SUCCESS
 from bentoml.exceptions import CLIException
+from bentoml._internal.cli.click_utils import _echo
```

Comment:
oo should we do relative import for this, considering this is under `_internal`?